projects.go

```diff
@@ -367,6 +367,27 @@ func (s *ProjectsService) CreateProject(opt *CreateProjectOptions, options ...Op
 	return p, resp, err
 }
 
+// ShareWithGroupOptions represents options to share project with groups
+//
+type ShareWithGroupOptions struct {
+	ID          *int              `url:"id" json:"id"`
+	GroupID     *int              `url:"group_id" json:"group_id"`
+	GroupAccess *AccessLevelValue `url:"group_access" json:"group_access"`
+	ExpiresAt   *string           `url:"expires_at" json:"expires_at"`
+}
+
+func (s *ProjectsService) ShareProjectWithGroup(opt *ShareWithGroupOptions, options ...OptionFunc) (*Response, error) {
+	u := fmt.Sprintf("projects/%d/share", *opt.ID)
```

Comment:
Don't use the `ID` from the options struct, but please put in the signature. And then before using it pass it through the `parseID` func so it becomes something like this:

```
	project, err := parseID(pid)
	if err != nil {
		return nil, err
	}
	u := fmt.Sprintf("projects/%s/share", url.QueryEscape(project))
```